commercetools-sdk-base/pom.xml

```diff
@@ -22,17 +22,17 @@
             <artifactId>sdk-http</artifactId>
             <version>${project.version}</version>
         </dependency>
-
+        <dependency>
+            <groupId>javax.money</groupId>
+            <artifactId>money-api</artifactId>
+            <version>1.0.1</version>
```

Comment:
Here we should introduce a new property `money-api.version` to our parent pom and reference it here with `${money-api.version}`.